services/api/handlers/projects.js

```diff
@@ -204,9 +205,15 @@ exports.get = {
 
 exports.patch = {
   update: function(request, reply) {
+    // hapi sets a pre-value to null if you pass reply an empty string :/
```

Comment:
There's nothing in the Hapi documentation that suggests you can change how the prerequisite reply interface works.